cypress/fixtures/prebid.ts

```diff
@@ -0,0 +1,60 @@
+export const bidderURLs = [
+	'https://ib.adnxs.com/ut/v3/prebid',
+	'https://htlb.casalemedia.com/cygnus**',
+	'https://hbopenbid.pubmatic.com/translator?source=prebid-client',
+	'https://prg.smartadserver.com/prebid/v1',
+	'https://rtb.openx.net/sync/prebid**',
+	'https://elb.the-ozone-project.com/openrtb2/auction',
+	'https://bidder.criteo.com/cdb**',
+	'https://pixel.adsafeprotected.com/services/pub**',
+	'https://ad.360yield.com/pb',
+];
+
+export const wins = {
+	criteo: {
+		url: 'https://bidder.criteo.com/cdb',
+		response: {
+			slots: [
+				{
+					slotid: 'aa828aaaeb0341b391df49b390e8a10c',
+					impid: 'dfp-ad--top-above-nav',
+					arbitrageid: '62cd3b90efd188239809bc56ec098900',
+					zoneid: 1171150,
+					cpm: 45,
+					currency: 'USD',
+					width: 970,
+					height: 250,
+					creative: '<h1 data-cy="test-creative">Hello</h1>',
+					video: false,
+					deal: '',
+					creativecode: '11096744',
+					is_rewarded: false,
+					paf_extension: { content_id: '' },
+				},
+			],
+			exd: {
+				slots: [
+					{
+						imp_id: 'aa828aaaeb0341b391df49b390e8a10c',
+						ad_unit_id: 'dfp-ad--top-above-nav',
+						zone_id: 1171150,
+						is_dsc: true,
+						enable_safeframe: false,
+					},
+				],
+			},
+			country_code: 'GB',
+			ext: {},
+		},
+		targeting: {
+			hb_format_criteo: 'banner',
+			hb_size_criteo: '970x250',
+			hb_pb_criteo: '45.00',
```

Comment:
Just curious about this, but do we need put a high CPM here to ensure Ad Manager doesn't select a higher CPM advert from the open market to return instead of the matching Prebid line item?

Are we always guaranteed that Ad Manager returns the Prebid line item in this case? I suppose nobody is going to pay >$45.00 for the targeting you get in a Cypress browser session 🤷

Reply:
Yeah that was my thinking, and it seems to work, but I wondered if it may affect analytics in some way?

You might be interested in joining Ravi, Emma and I, we're going to explore setting up a second GAM sandbox for integration testing at some point before the next meeting. So we have better control over this sort of stuff.

Reply:
Thanks for explaining Jake, yeah I'd definitely be up for exploring that with you all.

And I think you're right it'll affect analytics - Ad Manager won't know it's not a real $45.00 bid from Criteo that one. I assume this is a drop in the ocean for now, but might become an issue if we run it in CI against every push? Nothing a sandbox wouldn't solve though.